crypto/hash/auth.c

```diff
@@ -138,10 +136,11 @@ srtp_auth_type_test (const srtp_auth_type_t *at, const srtp_auth_test_case_t *te
                     srtp_octet_string_hex_string(test_case->key,
                                                  test_case->key_length_octets));
         debug_print(srtp_mod_auth, "data: %s",
-                    srtp_octet_string_hex_string(test_case->data,
-                                                 test_case->data_length_octets));
-        debug_print(srtp_mod_auth, "tag computed: %s",
-                    srtp_octet_string_hex_string(tag, test_case->tag_length_octets));
+                    srtp_octet_string_hex_string(
+                        test_case->data, test_case->data_length_octets));
+        debug_print(
+            srtp_mod_auth, "tag computed: %s",
+            srtp_octet_string_hex_string(tag, test_case->tag_length_octets));
```

Comment:
This will be a strange, the 3 debug statements will end up formatted differently ..